x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/security/action/user/GetUserPrivilegesResponseTests.java

```diff
@@ -231,15 +231,15 @@ private GetUserPrivilegesResponse.Indices randomIndices() {
             randomStringSet(8),
             Sets.newHashSet(
                 randomArray(
-                    3,
+                    1,
```

Comment:
Can we parameterize this since the regular indices should still work fine with `3`?

Reply:
Absolutely, good catch.